integration_tests/setup_db.py

```diff
@@ -253,10 +255,14 @@ def _setup_organisation(test_db: Session) -> int:
         "Test",
         org.id,
         "$2b$12$XXMr7xoEY2fzNiMR3hq.PeJBUUchJyiTfJP.Rt2eq9hsPzt9SXzFC",
-        is_active=True,
     )
+    _add_app_user(test_db, "test1@cpr.org", "TestInactive", org.id, is_active=False)
```

Comment:
Doesn't this user also have an empty hashed password?

Reply:
Technically, we test whether the user is active or not before we check the validity of their password against the hash.